job_spec/INSAR_ISCE_MULTI_BURST.yml

```diff
@@ -0,0 +1,87 @@
+INSAR_ISCE_MULTI_BURST:
+  required_parameters:
+    - reference
+    - secondary
+  parameters:
+    reference:
+      default:  '""'
+      api_schema:
+        type: array
+        minItems: 1
+        maxItems: 30
+        example:
+          - S1_136231_IW2_20200604T022312_VV_7C85-BURST
```

Comment:
The examples should probably have multiple bursts for this job type.

Reply:
Agreed. Perhaps we just add that as a TODO for when we release multi-burst InSAR to EDC prod?

Reply:
I added a TODO.